README.md

```diff
@@ -54,6 +54,21 @@ To get the frontend running locally:
  * Clone this repository
  * `yarn` to install all required dependencies
  * `yarn start` to start the local server
+ 
+ 
+ ## Prerequisites
+ 
+react-native
```

Comment:
Whole content is written in react native so we can easly transform this into native apps in future.